src/apiCalls.tsx

```diff
@@ -1 +1,45 @@
-export {}
+export const fetchNewsData = () => {
+  return (
+    fetch('https://api.nytimes.com/svc/topstories/v2/home.json?api-key=GKUzDD1VY9ssjZ1AGusX3ci6AeoXCaSr')
+    .then(response => checkResponse(response))
+    .then(data=> cleanNewsData(data.results))
+  )
+}
+
+const checkResponse = (response: any) => {
+  if (!response.ok) {
+    throw new Error(`${response.status} Error`)
+  }
+  return response.json()
+}
+
+interface Multimedia {
```

Comment:
Should these go at the top of the file?

Reply:
Not sure the best practice, but so far I've seen all interfaces declared at the top of the file.